autogen/main.tf.tmpl

```diff
@@ -267,10 +267,10 @@ resource "google_compute_firewall" "default-hc" {
   target_service_accounts = length(var.target_service_accounts) > 0 ? var.target_service_accounts : null
 
   dynamic "allow" {
-    for_each = var.backends
+    for_each = var.health_checked_backends
```

Comment:
Should be `local.health_checked_backends` not `var`

Reply:
yes, thank you